### ISSUE TITLE
[API] To add support of users tokens at least through graphql interface

### ISSUE THREAD

**Issue by RomanGalochkin** [foretold-app/foretold#693]

**Comment by OAGr** [comment]
Can you describe this more? I thought it was kind of working yesterday, in the graphql explorer (though this was a short-term token). Would this story need UI changes?

**Comment by RomanGalochkin** [comment]
It is like a question. The app has only one way of  authorization through "auth0.com". But what if I want to use "graphql" interface? At least I need my own token, but that token is too long and to get it I need to open application.

**Comment by OAGr** [comment]
What is this feature proposing? That we have a page in our application to get a custom,long-term token?

**Comment by RomanGalochkin** [comment]
Yes, exactly but first of all just a way to get user token through graphql interface.

**Comment by OAGr** [comment]
ok, that works
